bin/rake

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```

Comment:
Duplicate "standard" in "Add standard standard rake tasks and bin/setup" commit message.